examples/models/metadata/graph-metadata/input-transformer.yaml

```diff
@@ -9,26 +9,40 @@ spec:
   - componentSpecs:
     - spec:
         containers:
-        - image: seldonio/metadata-generic-node:0.2
+        - image: seldonio/metadata-generic-node:0.3
           name: node-input-transformer
           env:
           - name: MODEL_METADATA
             value: |
-              {"name": "node-input-transformer",
-               "versions": ["generic-node/v0.2"],
-               "platform": "seldon",
-               "inputs": [{"name": "input", "datatype": "BYTES", "shape": [1, 10]}],
-               "outputs": [{"name": "output", "datatype": "BYTES", "shape": [1, 20]}]}            
-        - image: seldonio/metadata-generic-node:0.2
+              ---
+              name: node-input-transformer
+              versions: [ generic-node/v0.3 ]
+              platform: seldon
+              inputs:
+              - messagetype: tensor
+                schema:
+                  names: [transformer-input]
```

Comment:
Would it not be here name and shape?

Reply:
I skipped `shapes` to make the graph metadata example notebook more readable. 
Can re-add them.

Reply:
Yeah I think otherwise it may be confusing for people that read it, may think that's what they should add

Reply:
Added shapes and improved naming in whole notebook.